### DISCUSSION TITLE
ios When filament trims a model, when a camera is pointed at half of the model, the whole model will be truncated and not displayed. Is there any relevant interface to avoid this truncation function

### DISCUSSION THREAD

**Discussion by guoaobao** [google/filament#6743] [Q&A]
When filament trims a model, when a camera is pointed at half of the model, the whole model will be truncated and not displayed. Is there any relevant interface to avoid this truncation function

**Comment by romainguy** [comment]
Culling is based on the bounding box you provide. This sounds like your bounding box doesn't enclose the entire model.
